test/protocol_test.py

```diff
@@ -1377,6 +1377,20 @@ def test_droplet_size(self, dummy_protocol):
                 "1.31:microliter",
             )
 
+    def test_propagate_properties(self, dummy_protocol):
+        p = dummy_protocol
+        p.propagate_properties = True
+        echo = p.ref("echo", None, "384-echo", discard=True)
+        dest = p.ref("dest", None, "384-echo", discard=True)
+        echo.well(0).add_properties({"test_well_property": True})
+        p.acoustic_transfer(
+            echo.well(0).set_volume("2:microliter"),
+            dest.well(0),
+            "50:nanoliter"
+        )
+        echo_prop = echo.well(0).properties["test_well_property"]
```

Comment:
I didn't know you could write it like this! I thought it had to be `well.properties.get["key"]`

Reply:
the `properties` attribute is a dictionary! you can use either `.get("key_name")` or `["key_name"]`

there's some implicit value over `get` as it returns `None` instead of raising a KeyError, but this value is dependent on your use case

Reply:
and I didnt realize you can run .get to avoid keyerrors 😂 we're learning _together_ @cj-bonilla